openapi/tools/merge_openapi.sh

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env bash
```

Comment:
I bet you could do this in the Gradle task and lose this file.

Reply:
I did not see how I could activate the virtual environment and run the script from within Gradle. The standard Exec task has a single command.

Reply:
You can use `doLast` to get another task in there, or do a `"my command".execute()`. https://stackoverflow.com/a/37520139/611672

Reply:
Those commands will not guaranteed run in the same shell, so the sourcing to set the venv will not be there in the second command.